### ISSUE TITLE
Im having a lot of trouble (confused)

### ISSUE THREAD

**Issue by BarynnD** [DRE2N/DungeonsXL#170]
Im using this plugin for my server and im trying to setup a dungeon for me and my friends to do, so far i have made a little room with new party and leave party signs, aswell as a portal that doesnt do anything, Idk how to link the portal to the dungeon btw.
But anyways what is happening is that im having trouble setting up a lobby, im not sure how these lobbies work because all i see in the pictures are [Lobby] and that doesnt do anything when I do it.
Basically when i try to do a sign nothing is working right, im pretty sure everything is installed properly.
Please help!

**Comment by Sataniel98** [comment]
Hi,

first of all, if you've got a dungeon portal, you're done. There is no need to link it to a dungeon. The group signs will handle everything else.

All you see is "[lobby]" because all you need is "[lobby]" :) Just place a sign wherever you want in your edit world.

**Comment by BarynnD** [comment]
Ok so if ive made the portal (with the wooden sword) I am done? I dont see how that would link the portal to the dungeon, And when they join a group what would they do after? Walk through the portal? and also on the lobby thing, so its just suppose to say lobby, because idk if its suppose to be interactable because if it is, it isn't. And also one more thing, when the players go through the portal with their party, and go into the lobby of the dungeon how do they enter the dungeon? Is there a special sign im suppose to use to start? I believe I saw something like [Start].

Thanks for the help in advance!

**Comment by Sataniel98** [comment]
You don't see how this would link the portal to the dungeon because it doesn't - you just add a group sign with the information about the dungeon it refers to. After that, you may use any dungeon portal. As explained in the wiki, the lobby sign marks a spawn point. Signs are usually not "interactable". Just a few ones have a built-in interact trigger.

See here for more information:

https://github.com/DRE2N/DungeonsXL/wiki/signs#triggers

https://github.com/DRE2N/DungeonsXL/wiki/signs#edit-signs
